plugins/llm-wiki/tests/resolver_smoke.sh

```
@@ -0,0 +1,59 @@
+[ "$OUT" = "$TMP/cache2/mkt/llm-wiki/2.5.1" ] && pass "incomplete 2.9.0 skipped, picked complete 2.5.1" || die "incomplete-version skip ($OUT)"
+
+# Nothing resolves: must abort nonzero
+if (cd "$TMP/foreign" && env -u CLAUDE_PLUGIN_ROOT -u HERMES_HOME CODEX_PLUGIN_CACHE="$TMP/none" bash "$BW") 2>/dev/null; then die "should abort with no root"; else pass "bootstrap: aborts when unresolved"; fi
+
+# review #4 — PLUGIN_ROOT is DEFINED and resolves in all 5 llm-wiki skills.
+for sk in bootstrap-wiki query-wiki ingest-finding lint-wiki migrate-wiki; do
+  SKILL="$PLUGIN_DIR/skills/$sk/SKILL.md"
+  grep -q 'PLUGIN_ROOT=' "$SKILL" || { die "$sk: no PLUGIN_ROOT definition"; continue; }
+  R=$(runner_for "$SKILL")
+  OUT=$(cd "$TMP/foreign" && env -u CLAUDE_PLUGIN_ROOT -u HERMES_HOME CODEX_PLUGIN_CACHE="$TMP/cache" bash "$R" 2>/dev/null)
+  { [ -n "$OUT" ] && [ -e "$OUT" ]; } && pass "$sk: PLUGIN_ROOT defined & resolves -> $OUT" || die "$sk: PLUGIN_ROOT unresolved ($OUT)"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)